docs/API.md

```diff
@@ -576,3 +578,31 @@ performance penalty.
 
 Returns an empty string. This function is used when the `timestamp` option
 is set to `false`.
+
+<a id="metadata"></a>
+# Metadata
+
+A destination stream can have a property `Symbol.for('needsMetadata')`
+to indicate that for every log line that is written, it wants set to
```

Comment:
It's the current level/msg/logger etc.
when write is called with `log.debug('foo')`, `lastLevel`  will be `20`, `lastMsg` will be `'foo'` and so on.